docs/API/v3/registration-base-url-resource.md

```diff
@@ -0,0 +1,282 @@
+metadata can be fetched using the `RegistrationsBaseUrl` resource found in the [service index](service-index.md).
+
+The collection of the documents found under `RegistrationsBaseUrl` are often called "registrations" or "registration
+blobs". The set of documents under a single `RegistrationsBaseUrl` is referred to as a "registration hive". A
+registration have contains all metadata about every package available on a package source.
+
+## Versioning
+
+There have been three versions of the `RegistrationsBaseUrl` released. All versions of the `RegistrationsBaseUrl` has
+the same general API surface area. However, there have been changes in the `Content-Encoding` and the support of SemVer
+2.0.0 packages. For more information about SemVer 2.0.0,
+[SemVer 2.0.0 protocol](https://github.com/NuGet/Home/wiki/Semver-2.0.0-Protocol) design.
+
+NuGet.org has three registration hives available for various client versions.
+
+### Initial hive
+
+The `@type` values for the initial release of the `RegistrationsBaseUrl` are:
+
+ - `RegistrationsBaseUrl`,
+ - `RegistrationsBaseUrl/3.0.0-beta`
+ - `RegistrationsBaseUrl/3.0.0-rc`
+
+These registrations are not compressed (meaning they use an implied `Content-Encoding: identity`). SemVer 2.0.0
+packages are **excluded** from this hive.
+
+### Gzipped hive - February 2016
+
+The `@type` value for the second release of the `RegistrationsBaseUrl` is:
+
+ - `RegistrationsBaseUrl/3.4.0`
+
+These registrations are compressed using `Content-Encoding: gzip`. SemVer 2.0.0 packages are **excluded** from this
+hive.
+
+### SemVer 2.0.0 hive - April 2017
+
+The `@type` values for the third release of the `RegistrationsBaseUrl` is:
+
+ - `RegistrationsBaseUrl/3.6.0`
+ - `RegistrationsBaseUrl/Versioned`
+   - `clientVersion` of `4.3.0-alpha`
+
+These registrations are compressed using `Content-Encoding: gzip`. SemVer 2.0.0 packages are **included** in this hive.
+
+## Base URL
+
+The base URL for the following APIs is the value of the `@id` property associated with the aforementioned
+resource `@type` values. In the following document, this placeholder URL will be used:
+
+```
+https://example/registration/
+```
+
+## HTTP methods
+
+All URLs found in the registration resource only support the HTTP method `GET`.
+
+## Registration index
+
+The registration resource groups package metadata by package ID. It is not possible to get data about more than one
+package ID at a time. This resource provides no way to discover package IDs. Instead the client is assumed to already
+know the desired package ID. Available metadata about each package version varies by server implementation. The package
+registration blobs have the following hierarchical structure:
+
+- **Index**: the entry point for the package metadata, shared by all packages on a source with the same package ID.
+- **Page**: a grouping of package versions. The number of package versions in a page is defined by server implementation.
```

Comment:
I would mention client behaviour here though: observing it has learned that *not* paging means client memory goes through the roof. Paging too agressively means too many HTTP requests over the wire. (see also https://github.com/NuGet/Home/issues/4448 for more info about large pages/large hives)

What is the size NuGet.org uses? I'd recommend this one for other implementors as well.

Reply:
Fixed.